### PR TITLE
Send spaceRef to identify space

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/10108
- Fixes https://github.com/owncloud/web/issues/10107
- ## Motivation and Context

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/helpers/share/link.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ export const copyQuicklink = async (args: CopyQuickLink) => {
 
   const linkSharesForResource = await clientService.owncloudSdk.shares.getShares(resource.path, {
     share_types: ShareTypes?.link?.value?.toString(),
+    spaceRef: `${resource.storageId}!${resource.nodeId}`,
```
Comment (kulmann) [inline packages/web-pkg/src/helpers/share/link.ts:83]: not possible to use `resource.id` or `resource.fileId`?

Comment (JanAckermann) [reply packages/web-pkg/src/helpers/share/link.ts:83]: I checked with Jörn, we think that's not possible. But I can check again

Comment (JanAckermann) [reply packages/web-pkg/src/helpers/share/link.ts]: You are right, works with fileId, this IDs driving me nuts 🙈

**Review by kulmann — APPROVED** [review]
💪

**Pull request merged**